test/edu/stanford/dlss/was/TestWasapiDownloaderSettings.java

```diff
@@ -2,34 +2,89 @@
 
 import java.io.IOException;
 
+import org.apache.commons.cli.ParseException;
+
+import static org.hamcrest.CoreMatchers.*;
 import static org.junit.Assert.*;
 import org.junit.*;
 
-import edu.stanford.dlss.was.WasapiDownloader;
-import edu.stanford.dlss.was.WasapiDownloaderSettings;
-
 public class TestWasapiDownloaderSettings {
 
-    @Before
-    public void setUp() {
-    }
+  @Before
+  public void setUp() {
+  }
```

Comment:
another empty @Before that could be deleted (and I still don't care if it is left in the PR)